integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/recipe/RecipeTestAction.java

```diff
@@ -0,0 +1,53 @@
+package com.sequenceiq.it.cloudbreak.newway.action.recipe;
+
+import static com.sequenceiq.it.cloudbreak.newway.log.Log.log;
+import static com.sequenceiq.it.cloudbreak.newway.log.Log.logJSON;
+import static java.lang.String.format;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+import com.sequenceiq.it.cloudbreak.newway.entity.recipe.RecipeTestDto;
+
+public class RecipeTestAction {
```

Comment:
It is not really an action, because it does not implement Action<RecipeTestDto>. It also has too many methods. 
Can you please create the separate recipe actions, and make this class a `client` like 
`com/sequenceiq/it/cloudbreak/newway/client/LdapConfigTestClient.java`?

Reply:
@attilapalfi92 could you please raise a Jira ticket to resolve all the related action classes whose are not extending the Action class? thank you!

Reply:
I will not raise separate jiras for every wrong action classes :D There is (or will be) a jira item for refactoring the refactored tests. Because the whole thing still smells like shit sadly.

Reply:
of course, you don't have to create tickets for all of them, just one in enough, indicating that we should fix all those places (no listing necessary). later on, this ticket will be linked to the refactor, but if we wait, after a couple of days nobody will remember

Reply:
thanks 😀

Reply:
you're proactive as usual 😀 we have a lot to thanks

Reply:
as we talked in Slack, the naming convention is TestAction/TestDto/TestAssertion/TestClient.
** TestAction: collection of actions
** TestAssertion: collection of assertions
** TestDto: wrapper class for request/response object
** TestClient: any other operation on the TestDto

Reply:
I cannot agree with this. TestAction should be one single action. A collection of actions should be provided by the client. This is how it makes sense.